conf/ledgersmb-nginx.template

```diff
@@ -0,0 +1,63 @@
+##
+# LedgerSMB server configuration
+#
+server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+
+        #Set to LSMB installed directory
+        root [ROOT];
+
+        index login.pl;
+
+        #Adjust to your liking
+        #server_name ubuntu-vm;
+
+	#Try the asked file directly, pass to Plack if not found
+        location / {
+            try_files $uri @plack;
+        }
+
+        location @plack {
+            proxy_set_header Host $http_host;
+            proxy_set_header X-Forwarded-Host $http_host;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_pass http://localhost:5000; # Adjust to Plack service
+        }
+
+        #Uncomment the next lines for monitoring
+        #access_log /var/log/nginx/ledgersmb.access.log main;
+        #error_log /var/log/nginx/ledgersmb.error.log error;
+
+        #Uncomment & adjust the next 4 lines for access control
+        #allow 127.0.0.1;
+        #allow ::1;
+        #allow 192.168.30.0/24;
+        #deny all;
+
+        # Deny external access to configuration and other nasty places.
+        location ~ \.conf$ { deny all; }
+        location ~ users { deny all; }
+        location ~ bin { deny all; }
+        location ~ blib { deny all; }
+        location ~ lib { deny all; }
+        location ~ tools { deny all; }
+        location ~ utils { deny all; }
+        location ~ spool { deny all; }
+        location ~ templates { deny all; }
+
+        #On Debian/Ubuntu 1.4 packages, css was on user control outside of lsmb directory
+        #location ~ lsmbcss {
+        #    alias /var/lib/ledgersmb/css/;
```

Comment:
That's why it is commented out. We do not want to deny access to css so
there is no need such a block in the config file.

Le 2016-04-22 15:51, Erik Huelsmann a écrit :
>
> In conf/ledgersmb-nginx.template
> <https://github.com/ehuelsmann/LedgerSMB/pull/240#discussion_r60793903>:
>
> > +        #deny all;
> > +
> > +        # Deny external access to configuration and other nasty places.
> > +        location ~ \.conf$ { deny all; }
> > +        location ~ users { deny all; }
> > +        location ~ bin { deny all; }
> > +        location ~ blib { deny all; }
> > +        location ~ lib { deny all; }
> > +        location ~ tools { deny all; }
> > +        location ~ utils { deny all; }
> > +        location ~ spool { deny all; }
> > +        location ~ templates { deny all; }
> > +
> > +        #On Debian/Ubuntu 1.4 packages, css was on user control outside of lsmb directory
> > +        #location ~ lsmbcss {
> > +        #    alias /var/lib/ledgersmb/css/;
>
> Hmm. Actually, I don't know. I don't think so. But I'd rather leave
> that to the packager and have something here which works for those
> installing from source (of which we still seem to have many).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ehuelsmann/LedgerSMB/pull/240/files/a6636f6dc8d0aad08544168b52862b70c41ba586#r60793903>
>